bitbots_body_behavior/bitbots_body_behavior/body_behavior.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, node:Node):
         node.create_subscription(GameState, "gamestate", blackboard.gamestate.gamestate_callback, qos_profile=1, callback_group=callback_group)
         node.create_subscription(TeamData, "team_data", blackboard.team_data.team_data_callback, qos_profile=1, callback_group=callback_group)
         node.create_subscription(PoseWithCovarianceStamped, "pose_with_covariance", blackboard.world_model.pose_callback, qos_profile=1, callback_group=callback_group)
-        node.create_subscription(RobotArray, "robot_obstacles", blackboard.world_model.robot_obstacle_callback, qos_profile=1, callback_group=callback_group)
+        node.create_subscription(RobotArray, "robot_obstacles", blackboard.costmap.robot_obstacle_callback, qos_profile=1, callback_group=callback_group)
```

Comment:
Change topic to `robots_relative_filtered`.

Reply:
why?

Reply:
Because the topic was renamed/reimplemented when we migrated to ros2 and we forgot to change it here. So the dynamic robots in the costmaps don't work at the moment ^^